Dockerfile

```diff
@@ -19,6 +19,8 @@ FROM nginx
 
 COPY nginx.conf /etc/nginx/conf.d/configfile.template
 COPY --from=build-env /app/build/ /usr/share/nginx/html
+
 ENV PORT 8080
 
+
 CMD sh -c "envsubst '\$PORT' < /etc/nginx/conf.d/configfile.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
```

Comment:
shouldn't this contain a line referencing the env-config.template?
you can have a look at the iesi backend Dockerfile for inspiration